tests/generic_config_updater/test_syslog.py

```diff
@@ -0,0 +1,267 @@
+import logging
+import pytest
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.config_reload import config_reload
+from tests.generic_config_updater.gu_utils import apply_patch, expect_op_success_and_reset_check, expect_res_success, expect_op_failure
+from tests.generic_config_updater.gu_utils import generate_tmpfile, delete_tmpfile
+
+pytestmark = [
+    pytest.mark.topology('t0'),
+]
+
+logger = logging.getLogger(__name__)
+
+# This is restricted by sonic-syslog.yang
+SYSLOG_MAX_SERVER=-1
+
+SYSLOG_THRESHOLD=10
```

Comment:
>10 [](http://example.com/codeflow?start=17&length=2)

Add document about its unit, is it in seconds?

Reply:
I am not sure about this. From my test, it doesn't take much time to do that. So I just give it a short threshold.

Reply:
Sorry I mean the unit of 10? Is it seconds, not milliseconds or others?

Reply:
Its unit is second